helm/adelphi/templates/results-server.yaml

```diff
@@ -1,33 +1,34 @@
-apiVersion: v1
-kind: Pod
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
 metadata:
-  name: results-server
+  name: results
   namespace: {{ .Values.namespace }}
+  annotations:
+    "helm.sh/hook": pre-install
 spec:
-  containers:
-    - name: results
+  templates:
+  - name: results-server
+    daemon: true
+    container:
       image: nginx
       volumeMounts:
       - name: config
         mountPath: /etc/nginx/conf.d
       - name: results-pv
         mountPath: /results
-  initContainers:
-  - name: clear-index
-    image: alpine
-    command:
-      - sh
-      - -c
-    args:
-      # remove sample index created by the NFS server
-      - rm /results/index.html
-    volumeMounts:
-    - name: results-pv
-      mountPath: /results
-  volumes:
-  - name: config
-    configMap:
-      name: results-server-configmap
-  - name: results-pv
-    persistentVolumeClaim:
-      claimName: results-pvc
\ No newline at end of file
+    initContainers:
+    - name: clear-index
+      image: alpine
+      command:
+        - sh
+        - -c
+      args:
+        # remove sample index created by the NFS server
+        - rm /results/index.html
```

Comment:
I was seeing errors with this due to /results/index.html, perhaps just on an initial run before anything had been generated (I'm assuming the persistent volume would address that for future runs).  I think just switching this to "rm -f /results/index.html" would be adequate to address it.

Reply:
Will address this in small follow-up PR